osu.Game.Rulesets.Osu/Mods/OsuModGrow.cs

```diff
@@ -15,6 +17,14 @@ internal class OsuModGrow : OsuModObjectScaleTween
 
         public override string Description => "Hit them at the right size!";
 
-        protected override float StartScale => 0.5f;
+        [SettingSource("Starting size modifier", "The object starting size modifier")]
+        public override BindableNumber<float> StartScale { get; } = new BindableFloat
+        {
+            MinValue = 0f,
+            MaxValue = 0.99f,
+            Default = 0.5f,
+            Value = 0.5f,
+            Precision = 0.01f,
```

Comment:
not sure I like this having a `0.01` step and being from `0-0.99` too much. I kind of think having this work as an inverse of `OsuModShrink` (so allowing initial scales of 1/2, 1/3, 1/4, ..., 1/25) might be better. would like opinions, however.

Reply:
I agree that 0.01 is too small and it doesn't make much sense having it close to 1, mostly went with the values used by the HT/DT mods as they have the same precision and allows you to modify the speed by 0.99/1.01. 
If I have understood everything correctly the `OsuModGrow` already is an inverse of `OsuModDeflate`, as giving the grow mod the value 2 for example makes it deflate instead?